src/test-env.js

```diff
@@ -7,17 +7,10 @@ const nodeUrls = process.env.NODE_URLS
 const privateKeys = process.env.PRIVATE_KEYS
 const networkType = process.env.NETWORK_TYPE
 
-export const E2eTestTags = [
-    'solo',
-    'default-private',
-    'testnet',
-    'mainnet',
-] as const
-
-export type E2eTestTag = (typeof E2eTestTags)[number]
+export const E2eTestTags = ['solo', 'default-private', 'testnet', 'mainnet']
```

Comment:
just a q - did we also had all here ?

Reply:
Nope, thats a param we can pass as an e2e tag